anubis-management-api/src/main.py

```diff
@@ -2,9 +2,27 @@
 from tenants import routers as t
 from policies import routers as p
 
+from fastapi.middleware.cors import CORSMiddleware
+
+import os
 
 app = FastAPI()
 
+allowed_origins = os.environ.get(
+    'CORS_ALLOWED_ORIGINS', "*").replace(" ", "").split(";")
+allowed_methods = os.environ.get(
+    'CORS_ALLOWED_METHODS', "*").replace(" ", "").split(";")
+allowed_headers = os.environ.get(
+    'CORS_ALLOWED_HEADERS', "*").replace(" ", "").split(";")
+
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=allowed_origins,
+    allow_credentials=True,
```

Comment:
allow_credentials - Indicate that cookies should be supported for cross-origin requests. Defaults to False. Also, allow_origins cannot be set to ['*'] for credentials to be allowed, origins must be specified.

https://fastapi.tiangolo.com/tutorial/cors/#use-corsmiddleware

Reply:
for now let's leave it as it is, but probably it would be good to configure this as well